cluster-api/src/main/java/io/aiven/klaw/clusterapi/services/AivenApiService.java

```diff
@@ -38,7 +38,7 @@ public Map<String, String> createAcls(ClusterAclRequest clusterAclRequest) {
     String projectName = clusterAclRequest.getProjectName();
     String serviceName = clusterAclRequest.getServiceName();
 
-    LinkedHashMap<String, String> permissionsMap = new LinkedHashMap<>();
+    Map<String, String> permissionsMap = new HashMap<>();
     permissionsMap.put("topic", clusterAclRequest.getTopicName());
     permissionsMap.put("permission", clusterAclRequest.getPermission());
     permissionsMap.put("username", clusterAclRequest.getUsername());
```

Comment:
Just an idea for future improvements (not necessary for this PR)
since we are the master of this map on both ends, could we replace it with `EnumMap` and create a corresponding enum?
Then we could get benefit of type safety and speed

Reply:
@snuyanzin agree. But in this case, it doesn't fit I guess, as this map part of the request goes to the REST call.